### PR TITLE
Prize and PrizeItems changes

### PR DESCRIPTION
**Pull request by amolskh**
Preview URL localhost:8080/#/PrizeExamples

Let me know if any other changes are required

Regards

### CODE REVIEW

**Comment by vikasrohit** [comment]
Linking to the issue #52

File: components/Prize/PrizeExamples.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+import React from 'react'
+import PrizeItems from './PrizeItems'
+
+/** Temporarly div Style can be replaced by tool tip componenet **/
+const divStyle = {
```
Comment (vikasrohit) [inline components/Prize/PrizeExamples.jsx]: Please extract css to separate scss file.

File: components/Prize/PrizeItem.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+require('./PrizeItem.scss')
+
+import React from 'react'
+
+const PrizeItem = (props) => {
```
Comment (vikasrohit) [inline components/Prize/PrizeItem.jsx]: We can use destructured properties here i.e `(props)` => `({title, subText})`

File: components/Prize/PrizeItem.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    font-weight: bold;
+    color:  #47474F;
+  }
+ 
+}
+
+.Prize {
+  .PrizeItem:nth-child(1) {
+    .PrizeCircle{
+      background: #FFD900;
```
Comment (vikasrohit) [inline components/Prize/PrizeItem.scss]: Please use variables for colors. If you don't find them in _tc-colors.scss (referenced from appirio-styles dependency), please put a variable in the top of the component's scss file.

File: components/Prize/PrizeItems.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+require('./PrizeItem.scss')
+
+import React from 'react'
+import PrizeItem from './PrizeItem'
+
+const PrizeItems = (props) => {
```
Comment (vikasrohit) [inline components/Prize/PrizeItems.jsx]: Again, please use destructured properties.

File: components/Prize/PrizeExamples.jsx (hunk 1 above)
Comment (amolskh) [inline components/Prize/PrizeExamples.jsx]: Do u mean css you to simulate toolip background should be extracted to separate css?

Comment (vikasrohit) [inline components/Prize/PrizeExamples.jsx]: Yes. Basically, any css should be in css files. For the actual component, there would be one css file and for the component examples, you can have another css file.

**Comment by amolskh** [comment]
Updated with changes for Sub track component and other fixes for both components

**Comment by vikasrohit** [comment]
@amolskh Can you please have separate PRs for different issues. It is for better tracking.

**Comment by vikasrohit** [comment]
Also, please use font-family as 'Roboto' instead of 'Roboto-Medium' or 'Roboto-Bold' and use font-weight with it to adjust for Medium, Regular or Bold.

**Comment by amolskh** [comment]
Ok i will change fonts.. It s not letter me create 2 seperate PR's from same fork.. I will try again if it lets me do

**Comment by vikasrohit** [comment]
Create separate branches for each issue in your forked repo and create PR from those branches to our master.

**Comment by amolskh** [comment]
ok will do

**Comment by vikasrohit** [comment]
@amolskh Can we delete this PR safely?

**Comment by amolskh** [comment]
yeah

**Pull request closed without merging**